### DISCUSSION TITLE
download error

### DISCUSSION THREAD

**Discussion by 4-Abdullah** [aclap-dev/video-downloadhelper#641] [Help and Q&A]
### Your question

{
  "version": "9.0.1.18",
  "target": "google",
  "channel": "stable",
  "page_url": "https://w1.nites.is/episode/peaky-blinders-1x1/#",
  "error": {
    "details": "SideDownload error: : [tcp @ 0000023f4979c940] Connection to tcp://gamn.v421c6e485f.site:443 failed: Error number -138 occurred\r\n[in#0 @ 0000023f47fd8a40] Error opening input: Error number -138 occurred\r\nError opening input file https://gamn.v421c6e485f.site/_v2-wwdo/12a3c523fd105800ed8c394685aeeb0bc22efd5c14e7f4f30c057baea93ece832257df1a4b6125fcfa38c35da05dee86aad28d46d73fc4e9d4e5a1381022a18937c511a05e48ea081586a3b967123946353dc065455e2cd1cb99bc5eccf52e907d47b90a0b71b7/h/aa4/a;15a38634f803584ba8926411d7bee906856cab0654b5bd.m3u8.\r\nError opening input files: Error number -138 occurred\r\n",
    "downloadable_id": "downloadable_6804502527516325",
    "error": "coapp_failure",
    "id": "coapp_failure_232419be-4b4a-40da-8732-9317793b171d"
  }
}

### Addon & CoApp version and details

version: 9.0.1.18
target: google
channel: stable
lang: en-GB
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36
{
    "used_sidebar_button": true,
    "used_popup_button": true,
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": false,
    "open_count_store": 15
}

**Comment by Mattmjs** [comment]
It looks to work in firefox, it was downloading before I stop it at 5%.
